app-backend/datamodel/src/main/scala/com/rasterfoundry/datamodel/ExportAssetType.scala

```diff
@@ -0,0 +1,34 @@
+package com.rasterfoundry.datamodel
+
+import cats.implicits._
+import io.circe._
+
+sealed abstract class ExportAssetType(val repr: String) {
+  override def toString = repr
+}
+
+object ExportAssetType {
+  case object SignedURL extends ExportAssetType("SIGNED_URL")
+  case object COG extends ExportAssetType("COG")
+
+  def fromString(s: String): Option[ExportAssetType] =
+    Option(s.toUpperCase) collect {
+      case "SIGNED_URL" => SignedURL
+      case "COG"        => COG
+    }
+
+  def unsafeFromString(s: String): ExportAssetType =
+    fromString(s).getOrElse(
+      throw new Exception(s"Unsupported export asset type string: $s")
+    )
+
+  implicit val exportAssetTypeEncoder: Encoder[ExportAssetType] =
+    Encoder.encodeString.contramap[ExportAssetType](_.toString)
+
+  implicit val exportAssetTypeDecoder: Decoder[ExportAssetType] =
+    Decoder.decodeString.emap { str =>
+      Either
+        .catchNonFatal(unsafeFromString(str))
+        .leftMap(_ => "ExportAssetType")
```

Comment:
You might want to provide a nicer error message here, since when bad strings are passed the user-facing message just says `DecodingFailure at .exportAssetTypes[0]: ExportAssetType`